pom.xml

```diff
@@ -276,26 +277,26 @@
 				<version>5.5.5</version>
 			</dependency>
 			<dependency>
-				<groupId>org.liquibase</groupId>
-				<artifactId>liquibase-core</artifactId>
-				<version>2.0.5</version>
-			</dependency>
-			<dependency>
-				<groupId>org.openmrs.liquibase.ext</groupId>
-				<artifactId>modify-column</artifactId>
-				<version>2.0.2</version>
+				<groupId>org.apache.lucene</groupId>
```

Comment:
Do we still need `lucene-backward-codecs`?

Reply:
Yes, I had commented on that the first time you had asked about updating the cache. I had a look at the codebase but did not find an easy way to implement that. I would rather do that in a separate ticket, this is more than a one-line change.

Reply:
This is a result of upgrading an OpenMRS database from 2.2.x to 2.4.x which is a manual test case. I guess the error that this test unearthed was there before and is not related to the upgrade to Liquibase as such.

Reply:
How does the error look like?  I have locally removed this dependency and compiled successfully. I also tested an upgrade from 1.11 to 2.4 without this dependency, but did not get into any form of trouble. 😊

Reply:
@WolfSchlegel any response to the above?

Reply:
Spring context fails to load, it is a serious failure

Reply:
How do i reproduce it? I have removed the dependency, compiled, and run without any problems.

Reply:
@dkayiwa That's interesting... Herbert Yiga's been looking into this issue [here](https://issues.openmrs.org/browse/TRUNK-5731) and claimed to reproduce it. I thought this was as easy as incrementing the constant [here](https://github.com/openmrs/openmrs-core/blob/a89743b5f9111ac2ffc59b82df1387d488ab08c4/api/src/main/java/org/openmrs/util/OpenmrsConstants.java#L599), which should force and index rebuild on startup. I haven't yet tried upgrading an install myself.

Reply:
@ibacher the upgrade from any version of the platform which has been running a lower version of lucene will always result into errors until one deletes the lucene folder in the application data directory. Which is not solved by including this dependency.

Reply:
Ah... I see. Yes, this dependency helps only in the narrow case of upgrading from 2.2 (or any version running Lucene 4) -> 2.4 (or any version running Lucene 5) without deleting the existing index.

I think, then, I'd be happy if we removed the dependency from this commit and worked on TRUNK-5731 (there's no reason deleting the index folder needs to be a manual upgrade step).

Reply:
I would suggest to remove that dependency in the context of TRUNK-5731.

Reply:
Come to think of it, the dependency cannot be removed in this commit as we would loose the ability to test upgrade scenarios. I updated TRUNK-5731 to ensure that the dependency is removed after fixing the bug.

Reply:
@WolfSchlegel i have been able to test upgrades without this dependency. What failure error message does this dependency solve for the upgrades?